packages/rancher-monitoring/generated-changes/patch/charts/hardenedKubelet/templates/pushprox-servicemonitor.yaml.patch

```diff
@@ -0,0 +1,54 @@
+--- charts-original/charts/hardenedKubelet/templates/pushprox-servicemonitor.yaml
++++ charts/charts/hardenedKubelet/templates/pushprox-servicemonitor.yaml
+@@ -13,6 +13,51 @@
+     params:
+       _scheme: [https]
+     {{- end }}
++    honorLabels: true
++{{- if .Values.metricRelabelings }}
++    metricRelabelings:
++{{ tpl (toYaml .Values.metricRelabelings | indent 4) . }}
++{{- end }}
++{{- if .Values.relabelings }}
++    relabelings:
++{{ toYaml .Values.relabelings | indent 4 }}
++{{- end }}
```

Comment:
Can these values be added under `.Values.serviceMonitor.metricsEndpoint.*` instead of at the root level?

Also, let's add this to the original pushprox chart instead of patching it.